frontend/test/metabase-visual/internal/static-viz.cy.spec.js

```diff
@@ -0,0 +1,25 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("visual tests > internal > static-viz", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("timeseries", () => {
+    cy.visit("/_internal/static-viz");
+    cy.findByText("Bar chart with timeseries data");
+    cy.findByText("Line chart with timeseries data");
+
+    cy.percySnapshot();
+  });
+
+  it("donut", () => {
+    cy.visit("/_internal/static-viz");
+    cy.scrollTo("bottom");
```

Comment:
I'm not happy with this, so maybe `frontend/src/metabase/internal/pages/StaticVizPage.jsx` needs some minor tweaks to have a proper anchor for each chart, and thereby we can scroll to that anchor in this test.

Reply:
Ha, it turns out that this is not necessary since Percy takes the snapshot of the entire page: https://percy.io/6a8dd5c9/metabase/builds/12247761/changed/690659721?browser=chrome&browser_ids=20&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1280&widths=1280